Assets/Shaders/Caustics.gdshader

```diff
@@ -66,23 +65,36 @@ varying vec3 positionWS;
 varying vec3 positionOS;
 varying vec3 screen_color;
 varying float bbox_mask;
+
 void fragment(){
 	float depth = texture(DEPTH_TEXTURE, SCREEN_UV).x;
-	vec3 ndc = vec3(SCREEN_UV, depth) * 2.0 - 1.0;
+	vec3 ndc;
+	if (OUTPUT_IS_SRGB){
```

Comment:
I confirmed with Clay John that OUTPUT_IS_SRGB is guaranteed to map to opengl/vulkan until 5.0 at least. This can be changed to an explicit renderer define if one is added in a future 4.x version.

Reply:
In 4.4 there's a more explicit way to do this with `CLIP_SPACE_FAR`, too.